File: javaVersion/Pedido.java
package javaVersion;

/**
 * Clase que representa un pedido en el restaurante
 */
public class Pedido {
    private int id;
    private String nombre_plato;
    
    public Pedido(int id, String nombre_plato) {
        this.id = id;
        this.nombre_plato = nombre_plato;
    }
    
    public int get_id() {
        return id;
    }
    
    public String get_nombre_plato() {
        return nombre_plato;
    }
    
    @Override
    public String toString() {
        return "Pedido #" + id + ": " + nombre_plato;
    }
}

File: python/pedido.py
"""
Módulo que define la clase Pedido.
"""

class Pedido:
    """Clase que representa un pedido con ID y nombre del plato"""
    
    def __init__(self, id_pedido, nombre_plato):
        self.id_pedido = id_pedido
        self.nombre_plato = nombre_plato
    
    def __str__(self):
        """Representación en cadena del pedido"""
        return f"Pedido #{self.id_pedido}: {self.nombre_plato}"